superset-frontend/src/explore/controls.jsx

```diff
@@ -63,9 +63,10 @@ import {
   getSequentialSchemeRegistry,
   legacyValidateInteger,
   validateNonEmpty,
-  ColumnOption,
 } from '@superset-ui/core';
 
```

Comment:
nit, remove the newline here (i need to see if i can programatically enforce it with prettier)